src/DMXSerial2.cpp

```diff
@@ -922,12 +1082,14 @@ void DMXSerialClass2::_saveEEPRom()
   eeprom.sig1 = 0x6D;
   eeprom.sig2 = 0x68;
   eeprom.startAddress = _startAddress;
-  strcpy (eeprom.deviceLabel, deviceLabel);
+  // This needs restricting to 32 chars (potentially no null), for backwards compatibility
+  strncpy(eeprom.deviceLabel, deviceLabel, DMXSERIAL_MAX_RDM_STRING_LENGTH);
```

Comment:
Should use a memcpy as both are known lengths

Reply:
Again #37 

I believe `abc\0def` is a valid RDM label, but only `abc` should be used, if we memcpy we'd needlessly write def to the EEPROM wouldn't we? Also it's max length is known, but not it's current length.